[2025-05-04T15:54:25.786+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-04T15:54:25.826+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.run_dbt_models manual__2025-05-04T15:47:55.314508+00:00 [queued]>
[2025-05-04T15:54:25.846+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.run_dbt_models manual__2025-05-04T15:47:55.314508+00:00 [queued]>
[2025-05-04T15:54:25.848+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-04T15:54:25.877+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): run_dbt_models> on 2025-05-04 15:47:55.314508+00:00
[2025-05-04T15:54:25.894+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=389) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-04T15:54:25.899+0000] {standard_task_runner.py:72} INFO - Started process 404 to run task
[2025-05-04T15:54:25.900+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'run_dbt_models', 'manual__2025-05-04T15:47:55.314508+00:00', '--job-id', '436', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmpc07sqbg4']
[2025-05-04T15:54:25.903+0000] {standard_task_runner.py:105} INFO - Job 436: Subtask run_dbt_models
[2025-05-04T15:54:25.982+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.run_dbt_models manual__2025-05-04T15:47:55.314508+00:00 [running]> on host 8280c28b7f0f
[2025-05-04T15:54:26.137+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='run_dbt_models' AIRFLOW_CTX_EXECUTION_DATE='2025-05-04T15:47:55.314508+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-04T15:47:55.314508+00:00'
[2025-05-04T15:54:26.139+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-04T15:54:26.140+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-04T15:54:26.141+0000] {logging_mixin.py:190} INFO - Current task name:run_dbt_models state:running start_date:2025-05-04 15:54:25.827871+00:00
[2025-05-04T15:54:26.142+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-04T15:54:26.142+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-04T15:54:26.144+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-05-04T15:54:26.146+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt_project/bank_reviews_dbt && dbt run --profiles-dir .']
[2025-05-04T15:54:26.170+0000] {subprocess.py:99} INFO - Output:
[2025-05-04T15:54:32.222+0000] {subprocess.py:106} INFO - [0m15:54:32  Running with dbt=1.9.4
[2025-05-04T15:54:32.951+0000] {subprocess.py:106} INFO - [0m15:54:32  Registered adapter: postgres=1.9.0
[2025-05-04T15:54:34.421+0000] {subprocess.py:106} INFO - [0m15:54:34  Found 5 models, 12 data tests, 433 macros
[2025-05-04T15:54:34.430+0000] {subprocess.py:106} INFO - [0m15:54:34
[2025-05-04T15:54:34.432+0000] {subprocess.py:106} INFO - [0m15:54:34  Concurrency: 1 threads (target='dev')
[2025-05-04T15:54:34.435+0000] {subprocess.py:106} INFO - [0m15:54:34
[2025-05-04T15:54:34.742+0000] {subprocess.py:106} INFO - [0m15:54:34  1 of 5 START sql table model review.dim_bank ................................... [RUN]
[2025-05-04T15:54:35.127+0000] {subprocess.py:106} INFO - [0m15:54:35  1 of 5 OK created sql table model review.dim_bank .............................. [[32mSELECT 31[0m in 0.38s]
[2025-05-04T15:54:35.133+0000] {subprocess.py:106} INFO - [0m15:54:35  2 of 5 START sql table model review.dim_branch ................................. [RUN]
[2025-05-04T15:54:35.303+0000] {subprocess.py:106} INFO - [0m15:54:35  2 of 5 OK created sql table model review.dim_branch ............................ [[32mSELECT 31[0m in 0.16s]
[2025-05-04T15:54:35.309+0000] {subprocess.py:106} INFO - [0m15:54:35  3 of 5 START sql table model review.dim_location ............................... [RUN]
[2025-05-04T15:54:35.540+0000] {subprocess.py:106} INFO - [0m15:54:35  3 of 5 OK created sql table model review.dim_location .......................... [[32mSELECT 38[0m in 0.23s]
[2025-05-04T15:54:35.546+0000] {subprocess.py:106} INFO - [0m15:54:35  4 of 5 START sql table model review.dim_sentiment .............................. [RUN]
[2025-05-04T15:54:35.708+0000] {subprocess.py:106} INFO - [0m15:54:35  4 of 5 OK created sql table model review.dim_sentiment ......................... [[32mSELECT 3[0m in 0.16s]
[2025-05-04T15:54:35.713+0000] {subprocess.py:106} INFO - [0m15:54:35  5 of 5 START sql table model review.fact_reviews ............................... [RUN]
[2025-05-04T15:54:35.881+0000] {subprocess.py:106} INFO - [0m15:54:35  5 of 5 OK created sql table model review.fact_reviews .......................... [[32mSELECT 171[0m in 0.16s]
[2025-05-04T15:54:35.919+0000] {subprocess.py:106} INFO - [0m15:54:35
[2025-05-04T15:54:35.921+0000] {subprocess.py:106} INFO - [0m15:54:35  Finished running 5 table models in 0 hours 0 minutes and 1.48 seconds (1.48s).
[2025-05-04T15:54:36.059+0000] {subprocess.py:106} INFO - [0m15:54:36
[2025-05-04T15:54:36.061+0000] {subprocess.py:106} INFO - [0m15:54:36  [32mCompleted successfully[0m
[2025-05-04T15:54:36.063+0000] {subprocess.py:106} INFO - [0m15:54:36
[2025-05-04T15:54:36.065+0000] {subprocess.py:106} INFO - [0m15:54:36  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2025-05-04T15:54:37.917+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-05-04T15:54:37.991+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-04T15:54:37.993+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=run_dbt_models, run_id=manual__2025-05-04T15:47:55.314508+00:00, execution_date=20250504T154755, start_date=20250504T155425, end_date=20250504T155437
[2025-05-04T15:54:38.040+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-04T15:54:38.041+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-04T15:54:38.042+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-04 15:47:55.335323+00:00
[2025-05-04T15:54:38.043+0000] {logging_mixin.py:190} INFO - Task hostname:8280c28b7f0f operator:BashOperator
[2025-05-04T15:54:38.072+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-04T15:54:38.138+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-04T15:54:38.142+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
